pkg/sources/gcppubsub/googleapis-serviceentry.yaml

```diff
@@ -24,3 +24,42 @@ spec:
     name: https
     protocol: HTTPS
   location: MESH_EXTERNAL
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: metadata-server-k8s
+spec:
+  hosts:
+  - metadata
```

Comment:
Why not put both names in this `hosts` list (and then you wouldn't need a second ServiceEntry)?

Reply:
@tcnghia created this sample. I'm not sure why, but I've just confirmed that the demo doesn't work when I combine the metadata and metadata.google.internal objects.

Reply:
Yes, it does not work when combined. He's filed some bugs against Istio.